<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>previewForm</class>
 <widget class="QDialog" name="previewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>436</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>436</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>436</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select a preview</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Take_Your_Pills_Video_Encoder_images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14px;</string>
  </property>
  <widget class="QPushButton" name="pushbuttonStop">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>385</y>
     <width>47</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/previewStopNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/previewStopHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/previewStopPressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbuttonPreviousStep">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>385</y>
     <width>47</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/previewPreviousStepNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/previewPreviousStepHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/previewPreviousStepPressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbuttonPreviousFrame">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>385</y>
     <width>47</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/previewPreviousFrameNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/previewPreviousFrameHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/previewPreviousFramePressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbuttonPause">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>385</y>
     <width>47</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/previewPauseNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/previewPauseHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/previewPausePressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Phonon::SeekSlider" name="phononSeekSlider">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>640</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal
{
	width: 20px;
	background-image: url(:/images/previewSeekSliderHandle.png);
}

QSlider::groove:horizontal
{
	background-image: url(:/images/previewSeekSliderBar.png);
}</string>
   </property>
   <property name="iconVisible">
    <bool>false</bool>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="Phonon::VideoPlayer" name="phononVideoPlayer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbuttonNextStep">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>385</y>
     <width>47</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/previewNextStepNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/previewNextStepHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/previewNextStepPressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbuttonPlay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>385</y>
     <width>47</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/previewPlayNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/previewPlayHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/previewPlayPressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbuttonNextFrame">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>385</y>
     <width>47</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/previewNextFrameNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/previewNextFrameHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/previewNextFramePressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbuttonChoose">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px;
	color: rgb(255, 255, 255);
	background-image: url(:/images/previewButtonNormal.png);
}

QPushButton:hover
{
	background-image: url(:/images/previewButtonHover.png);
}

QPushButton:pressed
{
	background-image: url(:/images/previewButtonPressed.png);
}</string>
   </property>
   <property name="text">
    <string>    Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTakeYourPillsLogo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Take_Your_Pills_Video_Encoder_images.qrc">:/images/previewLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>640</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	
	background-image: url(:/images/previewBackground.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelBackground</zorder>
  <zorder>pushbuttonStop</zorder>
  <zorder>pushbuttonPreviousStep</zorder>
  <zorder>pushbuttonPreviousFrame</zorder>
  <zorder>pushbuttonPause</zorder>
  <zorder>phononSeekSlider</zorder>
  <zorder>phononVideoPlayer</zorder>
  <zorder>pushbuttonNextStep</zorder>
  <zorder>pushbuttonPlay</zorder>
  <zorder>pushbuttonNextFrame</zorder>
  <zorder>pushbuttonChoose</zorder>
  <zorder>labelTakeYourPillsLogo</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Take_Your_Pills_Video_Encoder_images.qrc"/>
 </resources>
 <connections/>
</ui>
